plugins/bufbuild/connect-go/v1.4.0/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1.4
-FROM golang:1.19.4-bullseye AS build
+FROM golang:1.20.1-bullseye AS build
```

Comment:
Why is this being updated?

Reply:
Hehe, I did a quick search and replace based on semver, but picked up connect-go instead of chrusty-jsonschema

Reply:
need to be a bit more careful as we have a good matrix of versions now